framework/docker/complete/compose.yml

```diff
@@ -14,7 +14,7 @@ services:
     build:
       context: ${PROJECT_DIR:-.}
       dockerfile_inline: |
-        FROM flwr/serverapp:${FLWR_VERSION:-1.21.0}
+        FROM flwr/serverapp:${FLWR_VERSION:-1.22.0}
```

Comment:
Have we not updated the docker compose files/docs? they still use ServerApp and ClientApp.